packages/recipe-ui-theme-marketplace/src/index.ts

```diff
@@ -7,16 +7,42 @@ const marketplace = createTheme({
   },
   borderWidths: {
     'card-border': '$thin',
+    'button-secondary': '$thin',
+    'button-secondary-focus': '$wide',
   },
   colors: {
     'page-bg': 'white',
     'card-border': '$colors$gray150',
+
+    /* secondary button default */
+    'button-bg-secondary': '$white',
+    'button-border-secondary': '$blue600',
+    'button-text-secondary': '$blue600',
+
+    /* secondary button hover */
+    'button-bg-secondary-hover': '$blue200',
+    'button-border-secondary-hover': '$blue600',
+    'button-text-secondary-hover': '$blue600',
+
+    /* secondary button active */
+    'button-bg-secondary-down': '$blue300',
+    'button-border-secondary-down': '$blue600',
+    'button-text-secondary-down': '$blue800',
+
+    /* secondary button focus */
+    'button-bg-secondary-focus': '$blue200',
+    'button-border-secondary-focus': 'rgba(63, 97, 255, 0.3)',
+    'button-text-secondary-focus': '$blue600',
   },
   radii: {
     'flash-message-rounded': 0,
   },
   shadows: {
     card: 'none',
+    'button-secondary': '0px 1px 2px rgba(0, 0, 0, 0.12)',
+    'button-secondary-hover': '0px 1px 6px rgba(0, 0, 0, 0.16)',
+    'button-secondary-down': '0px 0px 3px rgba(0, 0, 0, 0.2)',
+    'button-secondary-focus-ring': '0 0 3px rgba(63, 97, 255, 0.3)',
```

Comment:
Changing this `box-shadow` for secondary buttons would give the appearance of a border so we don't have to deal with changing border widths which cause shifting of the element.

```suggestion
    'button-secondary-focus-ring': '0 0 0 3px rgba(180, 190, 237, 1)',
```
<img width="88" alt="Screen Shot 2022-04-29 at 1 28 54 PM" src="https://user-images.githubusercontent.com/5418735/165993939-d3717f10-4310-4b86-b86b-18b3e0e05efe.png">

Reply:
done!